packages/firebase_database/firebase_database_web/lib/src/interop/data_snapshot_interop.dart

```diff
@@ -5,11 +5,11 @@ part of firebase.database_interop;
 abstract class DataSnapshotJsImpl {
   external String get key;
 
-  external set key(String s);
-
   external ReferenceJsImpl get ref;
 
-  external set ref(ReferenceJsImpl r);
+  external dynamic /* string | num | null*/ get priority;
```

Comment:
Are those just for comment or is it interpreted by the interop?

Reply:
Just comments for understanding 👍